tests/helpers/fixtures.py

```diff
@@ -100,7 +100,10 @@ def check(self, expected):
         if self._result is self.UNSET:
             with self._qtbot.waitSignal(self.got_result, timeout=2000):
                 pass
-        assert self._result == expected
+        self._assert_result(self._result, expected)
+
+    def _assert_result(self, result, expected):
```

Comment:
I made this change because I wasn't getting printouts of the 'result' value when the assert was inlined for some reason. I think it's related to creating new CallbackCheckers maybe?

Reply:
Hmm... pytest requires [registering files](https://docs.pytest.org/en/latest/writing_plugins.html#pytest.register_assert_rewrite) which should have rewritten assertions if it's not in a `test_*` or `conftest.py` file.

However, I don't see how a function would help with that...

I think `CallbackChecker` should be moved to `tests/helpers/utils.py` anyways now that it's used outside of `fixtures.py`. After you did that, I think rewriting should also work automatically, as [that file is registered](https://github.com/qutebrowser/qutebrowser/blob/7cc3fb4a4ea7ec4b05129af53b29e44edfe9c276/tests/conftest.py#L32).